### ISSUE TITLE
Option to select specific articles to display

### ISSUE THREAD

**Issue by mikeriley131** [chrisj74/vue-rss-blog#3]
Not sure if this would be feasible, but we're looking for a solution that would allow us to display specific articles on a specific page. Is that possible? Everything about your project is really nice. Thank you for creating and maintaining it.

**Comment by chrisj74** [comment]
Hi, thanks for the suggestion. Interesting one. It would be hard to do this as the code has no way of knowing what the RSS feed will contain. Were you thinking of applying a filter or something?

In principle, you could do a basic string match on the title, but it would help to understand the use case you imagine for this.

**Comment by mikeriley131** [comment]
Yeah, our fallback was to filter by string match, but I was wondering if it would be feasible to add an option that was an array of post IDs or something along those lines.

**Comment by sif** [comment]
I wonder if we are able to change what goes on the RSS feed to begin with. I am not seeing any option to do that in Medium. At the time of this comment, I only see this in their documentation when I search for "rss": https://help.medium.com/hc/en-us/articles/214874118-RSS-feeds

**Comment by ethanosullivan** [comment]
It would be nice to set a number of posts to display. For example, _show the last 5 Medium articles_
